core/trino-main/src/main/java/io/trino/testing/TestingAccessControlManager.java

```diff
@@ -258,11 +276,22 @@ public Set<SchemaTableName> filterTables(SecurityContext context, String catalog
     @Override
     public void checkCanImpersonateUser(Identity identity, String userName)
     {
-        if (shouldDenyPrivilege(userName, userName, IMPERSONATE_USER)) {
-            denyImpersonateUser(identity.getUser(), userName);
+        // if allowImpersonationRoles is empty default behaviour is used
```

Comment:
Remove the comment

Reply:
Why? It is explains why it is done this way. Assuming we will stay with `Optional<Set>>`

Reply:
I see exactly the same in line below in java. I mean the comment is obvious. It is a nit btw.